components/dialog/dialog.js

```diff
@@ -107,6 +103,7 @@ class Dialog extends PropertyRequiredMixin(LocalizeCoreElement(AsyncContainerMix
 		this.critical = false;
 		this.describeContent = false;
 		this.fullHeight = false;
+		this.predicate = () => true;
```

Comment:
I think I'd suggest not exposing this on dialog until we actually come across a case where we need it. It's also a bit strange that it's a reactive property but has no impact on `render()` really.